docs/collecting-container-logs.md

```diff
@@ -0,0 +1,170 @@
+
+This feature is enabled by default. It can rarely cause problems by merging together lines which are supposed to be separate.
+In that case, feel free to disable it.
+
+### Setting source name and other built-in metadata
+
+It's possible to customize the built-in Sumo Logic metadata (like [source name][source_name] for example) for container logs:
+
+```yaml
+sumologic:
+  logs:
+    container:
+      ## Set the _sourceHost metadata field in Sumo Logic.
+      sourceHost: "%{k8s.pod.hostname}"
+      ## Set the _sourceName metadata field in Sumo Logic.
+      sourceName: "%{k8s.namespace.name}.%{k8s.pod.name}.%{k8s.container.name}"
+      ## Set the _sourceCategory metadata field in Sumo Logic.
+      sourceCategory: "%{k8s.namespace.name}/%{k8s.pod.pod_name}"
+      ## Set the prefix, for _sourceCategory metadata.
+      sourceCategoryPrefix: "kubernetes/"
+      ## Used to replace - with another character.
+      sourceCategoryReplaceDash: "/"
+```
+
+As can be seen in the above example, these fields can contain templates of the form `%{field_name}`, where `field_name` is the name
+of a resource attribute. Available resource attributes include [OpenTelemetry Kubernetes resource attributes][opentelemetry_k8s],
+in addition to the following:
+
+- `cluster`
+- `_collector`
+- `pod_labels_*` where * is the Pod label name
+
+### Filtering
+
+Logs can be excluded based on their container name, pod name, host, and namespace. This is done by providing a matching
+regular expression:
+
+```yaml
+sumologic:
+  logs:
+    container:
+      ## A regular expression for containers.
+      ## Matching containers will be excluded from Sumo. The logs will still be sent to logs metadata provider (otelcol).
+      excludeContainerRegex: ""
+      ## A regular expression for hosts.
+      ## Matching hosts will be excluded from Sumo. The logs will still be sent to logs metadata provider (otelcol).
+      excludeHostRegex: ""
+      ## A regular expression for namespaces.
+      ## Matching namespaces will be excluded from Sumo. The logs will still be sent to logs metadata provider (otelcol).
+      excludeNamespaceRegex: ""
+      ## A regular expression for pods.
+      ## Matching pods will be excluded from Sumo. The logs will still be sent to logs metadata provider (otelcol).
+      excludePodRegex: ""
+```
+
+For more advanced filtering logic, see [here](#advanced-filtering).
+
+### Modifying log records
+
+:construction: This needs `extraProcessors` for logs.
+
+### Persistence
+
+By default, the metadata enrichment service provisions and uses a Kubernetes PersistentVolume as an on-disk queue that guarantees
+durability across Pod restarts and buffering in case of exporting problems.
+
+This feature is enabled by default, but it only works if you have a correctly configured default `storageClass` in your cluster. Cloud
+providers will do this for you when provisioning the cluster. The only alternative is disabling persistence altogether.
```

Comment:
We should add this case to troubleshooting

Reply:
Created an issue for it: https://github.com/SumoLogic/sumologic-kubernetes-collection/issues/2770, don't know off the top of my head what the symptoms are.